source/configuration_manual/nfs.rst

```diff
@@ -4,178 +4,109 @@
 NFS
 ###
 
-NFS is commonly used in one of these ways:
-
-1. Dovecot is run in a single computer.
-
-2. Dovecot is run in multiple computers, users are redirected more or
-   less randomly to different computers.
+Dovecot is commonly used with NFS. However, Dovecot does **not** support
+accessing the same user simultaneously by different servers. That will
+result in more or less severe mailbox corruption. Note that this applies
+to all kinds of mailbox access, including mail deliveries.
```

Comment:
I would remove "kinds of" and just say "applies to all mailbox access, including mail delivery."